sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSGP.`, () => {
+  it("Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true when studentDataApplicationPDPPDStatus is true", async () => {
```

Comment:
minor: "Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true ***and*** studentDataApplicationPDPPDStatus is true"